auto_backup/models/db_backup.py

```diff
@@ -180,7 +180,7 @@ def action_backup(self):
                         with rec.sftp_connection() as remote:
                             # Directory must exist
```

Comment:
I think this comment could be removed in face of https://bitbucket.org/dundeemt/pysftp/src/1c0791759688a733a558b1a25d9ae04f52cf6a64/pysftp/__init__.py#lines-676